Source/Core/DolphinWX/Main.cpp

```diff
@@ -108,8 +107,16 @@ bool DolphinApp::OnInit()
   wxHandleFatalExceptions(true);
 #endif
 
+#ifdef _WIN32
+  AttachConsole(ATTACH_PARENT_PROCESS);
```

Comment:
This has a `BOOL` return value, it should probably only redirect output if it succeeds in getting a console? That might fix @sepalani's cygwin issue.